sensing/preprocessor/gnss/geo_pos_conv/src/geo_pos_conv.cpp

```diff
@@ -209,68 +214,75 @@ void geo_pos_conv::conv_llh2xyz(void)
   AW = 6378137.0;            // Semimajor Axis
   FW = 1.0 / 298.257222101;  // 298.257223563 //Geometrical flattening
 
-  Pe = (double)sqrt(2.0 * FW - pow(FW, 2));
-  Pet = (double)sqrt(pow(Pe, 2) / (1.0 - pow(Pe, 2)));
+  Pe = static_cast<double>(sqrt(2.0 * FW - pow(FW, 2)));
+  Pet = static_cast<double>(sqrt(pow(Pe, 2) / (1.0 - pow(Pe, 2))));
 
-  PA = (double)1.0 + 3.0 / 4.0 * pow(Pe, 2) + 45.0 / 64.0 * pow(Pe, 4) +
```

Comment:
I think these casts were unnecessary. First, the casts have higher precedence than arithmetic operators, so e.g. here it only applied to the 1.0 literal which is already a double. In the cases where it is applied to e.g. `sqrt`, it would be casted to double anyway because the result is stored in a double. Therefore please remove all static casts in this file as well.